lib/cli/package.json

```diff
@@ -68,7 +68,8 @@
     "@types/puppeteer-core": "^2.0.0",
     "@types/semver": "^7.2.0",
     "@types/shelljs": "^0.8.7",
-    "@types/update-notifier": "^0.0.30"
+    "@types/update-notifier": "^0.0.30",
+    "rimraf": "^3.0.2"
```

Comment:
I think you can use [`fs-extra`](https://github.com/jprichardson/node-fs-extra#methods) (which is already a dep) instead of adding `rimraf`